_posts/2023-08-29-xmlfeeds.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: "[CVE-2023-39643] Improper neutralization of SQL parameter in XML Feeds PRO module for PrestaShop"
+categories: modules
+author:
+- TouchWeb.fr
+- 202 Ecommerce
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,xmlfeeds"
+severity: "critical (9.8)"
+---
+
+In the module "XML Feeds PRO" (xmlfeeds) up to versions 3.8.2 from Bl Modules for PrestaShop, a guest can perform SQL injection in affected versions.
+
+
+## Summary
+
+* **CVE ID**: [CVE-2023-39643](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-39643)
+* **Published at**: 2023-08-29
+* **Platform**: PrestaShop
+* **Product**: xmlfeeds
+* **Impacted release**: <= 3.8.2 (3.9.8 fixed the vulnerability)
+* **Product author**: Bl Modules
+* **Weakness**: [CWE-89](https://cwe.mitre.org/data/definitions/89.html)
+* **Severity**: critical (9.8)
+
+## Description
+
+The method `SearchApiXml::Xmlfeeds()` has sensitive SQL calls that can be executed with a trivial http call and exploited to forge a SQL injection.
+
+**WARNING** : This exploit is actively used to deploy webskimmer to massively steal credit cards. 
```

Comment:
**WARNING** : This exploit is actively used to deploy a webskimmer to massively steal credit cards.